INSTALL.md

```diff
@@ -9,30 +9,50 @@ The architecture of kernelCI is split into three main components:
 * Frontend
 * Backend
 
-For more information on the architecture please visit:  http://wiki.kernelci.org/
+For more information on the architecture please visit: http://wiki.kernelci.org/
 
 ## Prerequisites
 
-* The host machine which will run the ansible scripts to remotely install the front end onto the target machine. 
-	* Git tools
-	* Ansible >= 2.0: http://docs.ansible.com/ansible/latest/intro_installation.html
-* The target machine where the kernelci infrastructure (front and/or back end) will be deployed 
-	* Supported OS: Debian (Jessie)
-	* ssh root access to the server
-	* Python >= 2.7.12
+Two machines:
+* The host machine which will run the ansible scripts to remotely install the back end onto the target machine.
+	* Host configuration prerequisites
+		* Git tools
+		* Ansible >= 2.0: http://docs.ansible.com/ansible/latest/intro_installation.html
+* The target machine where the kernelci infrastructure (front and back end) will be deployed
+	* Target configuration prerequisites
+		* Supported OS: Debian (Jessie and after), Centos >= 7
+		* ssh root access to the server
+		* Python >= 2.7.12
+
+* Ansible need to have a way to be root either with direct root connect or either with user+sudo/su, you need to choose which way will be used.
+	* For direct root access:
+		* add ansible_ssh_user=root after serverhostname in hosts
+	* For user+sudo
+		* add ansible_ssh_user=user become_method=sudo after serverhostname in hosts
+	* More informations on http://docs.ansible.com/ansible/latest/become.html
 
 ## Installing
 
+### Install order
+* You need to install the backend first
+* Then you need to generate tokens for frontend and set them in secrets.yml
+* Then install the frontend
+
 ### Get the source code
+For Deploying kernelci-frontend
 ```
 $ git clone https://github.com/kernelci/kernelci-frontend-config.git
 ```
 
 ### Configure Host
-Edit /etc/hosts - add new line:
-```
-<TARGET_IP> <TARGET_NAME>
-```
+* You need to choose the FQDN used for calling both the frontend and backend.
+  These FQDN must be set in the hostname variable. (default from group_vars/all is kernelci-frontend/kernelci-backend)
+  Thoses name could be different from the hostname of the host.
+
+* then add the choosen hostname in the hosts file
+	* Example:
+		* kernelci.mydns.com ansible_ssh_user=root
+		* kernelci.mydns.com ansible_ssh_user=admin become_method=sudo
```

Comment:
Here probably it would be better to add a code block, instead of having the bullet list, to give a better example of how the file should be written.

Reply:
ok